subprojects/core/src/integTest/groovy/org/gradle/groovy/scripts/GradleUtilDefaultImportDeprecationIntegrationTest.groovy

```diff
@@ -57,7 +61,24 @@ task noop{
         succeeds('noop')
 
         then:
-        outputContains("Using GradleVersion from the private org.gradle.util package without an explicit import is deprecated. Please either stop using these private classes (recommended) or import them explicitly at the top of your build file (not recommended). The implicit import will be removed in Gradle 5.0")
+        outputContains("build.gradle' is using GradleVersion from the private org.gradle.util package without an explicit import. Please either stop using these internal classes (recommended) or import them explicitly at the top of your build file. The implicit import is deprecated and will be removed in Gradle 5.0")
```

Comment:
Nit-pick: have you noticed the `"build.gradle'` is half-single-quote and half-double-quote?

Reply:
It's not, the double quote is for the String, the single quote is at the end of the file name. I didn't want to test for the full file name here.